stubs/AMQP.php

```diff
@@ -66,11 +66,16 @@
 
 /**
  * When publishing a message, the message must be routed to a valid queue. If it is not, an error will be returned.
+ * Note: This flag does not currently work in pecl-amqp, and WILL cause undefined behavior if you use it.
+ * @link https://bugs.php.net/bug.php?id=62786
+ * @link https://github.com/pdezwart/php-amqp/issues/23
  */
 define('AMQP_MANDATORY', 1024);
 
 /**
- * When publishing a message, mark this message for immediate processing by the broker. (High priority message.)
+ * When publishing a message, the message must be routed to a valid queue with an available consumer. If it is not, an
```

Comment:
There is [AMQP spec](https://www.rabbitmq.com/resources/specs/amqp-xml-doc0-9-1.pdf) (section 1.8.3.7.5) and also [RabbitMQ doc](https://www.rabbitmq.com/amqp-0-9-1-reference.html#basic.publish.immediate) which covers what this flag is for. If you use RabbitMQ, then check it [spec](https://www.rabbitmq.com/specification.html), it doesn't support all AMQP 0.9.1 features as-is, e.g. `immediate` flag is not supported. As RabbitMQ is not the only AMQP broker, I don't think that we should cover all broker-specific cases in stub comments.